helper/src/main/java/me/lucko/helper/command/AbstractCommand.java

```diff
@@ -30,21 +30,25 @@
 import me.lucko.helper.internal.LoaderUtils;
 import me.lucko.helper.utils.CommandMapUtil;
 import me.lucko.helper.utils.annotation.NonnullByDefault;
-
 import org.bukkit.command.CommandExecutor;
 import org.bukkit.command.CommandSender;
+import org.bukkit.command.TabCompleter;
 
 import javax.annotation.Nullable;
+import java.util.List;
 
 /**
  * An abstract implementation of {@link Command} and {@link CommandExecutor}
  */
 @NonnullByDefault
-public abstract class AbstractCommand implements Command, CommandExecutor {
+public abstract class AbstractCommand implements Command, CommandExecutor, TabCompleter {
 
-    protected @Nullable String permission;
-    protected @Nullable String permissionMessasge;
-    protected @Nullable String descritpion;
+    @Nullable
+    protected String permission;
+    @Nullable
+    protected String permissionMessasge;
+    @Nullable
+    protected String descritpion;
```

Comment:
It would be easier to analyze your intended changes if you'd remove arbitrary formatting changes, such as this.

If it bothers you (which, as a fellow anally-retentive format-nazi, I understand), I suggest creating another PR solely for formatting fixes.